spec/dap/proto/ldap_proto_spec.rb

```diff
@@ -0,0 +1,139 @@
+module Dap
+module Proto
+class LDAP
+
+require 'openssl'
+require_relative '../../../lib/dap/proto/ldap'
```

Comment:
I just saw this while browsing.  I believe the pattern is to have a `spec_helper.rb` that is required in all specs with `require 'spec_helper'` and that helper requires the entire library, which hopefully then requires all the other modules such that only `require 'dap'` is needed in the helper.  I tried to follow that in sonar-client here: https://github.com/rapid7/sonar-client/tree/master/spec